README.md

```diff
@@ -19,7 +19,16 @@ export PATH="$GOROOT/bin:$PATH"
 ```
 2. Fork aws-lc on Github
 ```
-git clone https://github.com/${YOUR_GITHUB_ACCOUNT_NAME}/aws-lc.git
+# Create your fork in GitHub if it doesn't exist and clone it
+git clone git@github.com:${GITHUB_USERNAME}/aws-lc.git && cd aws-lc
+git remote add upstream git@github.com:awslabs/aws-lc.git
+
+# Update your fork and branch if needed
+git checkout main
+git fetch upstream
+git merge upstream/main
```

Comment:
I do rebase in this step instead of merge. I don't know if it's worth mentioning.